test/integration/test.js

```diff
@@ -356,7 +356,12 @@ describe('integration', function() {
         function() {
           var ravenData = iframe.contentWindow.ravenData[0];
           assert.equal(ravenData.exception.values[0].type, undefined);
-          assert.equal(ravenData.exception.values[0].value, '[object Object]');
+
+          // #<Object> is covering default Android 4.4 and 5.1 browser
+          assert.match(
+            ravenData.exception.values[0].value,
+            /^\[object Object\]|#<Object>$/
```

Comment:
This regex needs a group to be what I think you want:

```js
/^(\[object Object\]|#<Object>)$/
```

Reply:
You're right, thanks!

![screen shot 2017-10-04 at 14 18 31](https://user-images.githubusercontent.com/1523305/31175269-03dbc3fc-a90f-11e7-9893-ccad51a0e2f8.png)
vs.
![screen shot 2017-10-04 at 14 18 24](https://user-images.githubusercontent.com/1523305/31175270-03dcb000-a90f-11e7-9b66-f385295d1821.png)